src/RCController/RCController.cpp

```diff
@@ -80,46 +93,53 @@ namespace tritonai::gkc
 
-            bool is_the_same_data = Map.throttle(busData[ELRS_THROTLE]) == _packet.throttle &&
-                Map.steering(busData[ELRS_STEERING]) == _packet.steering;
-
-            bool is_all_zero = abs(100*Map.throttle(busData[ELRS_THROTLE])) <= 5 && abs(100*Map.throttle(busData[ELRS_STEERING])) <= 5;
-
-            // Check if the values are the same as the previous ones
-            if (is_the_same_data && !is_all_zero)
-                continue; // Stop if the values are the same
+            // std::cout << "Bus data: " << (int)(100*busData[0]) <<
+            //     " " << (int)(100*busData[1]) <<
+            //     " " << (int)(100*busData[2]) <<
+            //     " " << (int)(100*busData[3]) <<
+            //     " " << (int)(100*busData[4]) <<
+            //     " " << (int)(100*busData[5]) <<
+            //     " " << (int)(100*busData[6]) <<
+            //     " " << (int)(100*busData[7]) <<
+            //     " " << (int)(100*busData[8]) <<
+            //     std::endl;
 
             // Stop if the emergency stop is not active
             bool temp_active = Map.is_active(
                 busData[ELRS_EMERGENCY_STOP_LEFT],
                 busData[ELRS_EMERGENCY_STOP_RIGHT]
             );
 
-            if(is_all_zero){
+            if(!temp_active){
                 _packet.throttle = 0.0;
                 _packet.steering = 0.0;
-                _packet.brake = 0.0;
+                _packet.brake = Map.throttle_ratio(busData[ELRS_RATIO_THROTTLE]);
                 _packet.is_active = temp_active;
-                _packet.autonomy_mode = Map.getAutonomyMode(
-                    busData[ELRS_TRI_SWITCH_RIGHT]
-                );
                 _packet.publish(*_sub);
                 continue;
             }
 
-            // if(temp_active == false && _packet.is_active == false) continue; // Stop if the values are the same
+            bool keep_constant_thr = Map.keep_constant_thr(busData[ELRS_HOLD_THROTTLE]);
+            bool is_all_zero = abs(100*Map.throttle(busData[ELRS_THROTLE])) <= 5 && abs(100*Map.throttle(busData[ELRS_STEERING])) <= 5;
+
 
-            if(!temp_active){
+            if(is_all_zero && !keep_constant_thr){
                 _packet.throttle = 0.0;
                 _packet.steering = 0.0;
-                _packet.brake = 0.2;
+                _packet.brake = 0.0;
                 _packet.is_active = temp_active;
+                _packet.autonomy_mode = Map.getAutonomyMode(
+                    busData[ELRS_TRI_SWITCH_RIGHT]
+                );
                 _packet.publish(*_sub);
                 continue;
             }
 
+            if(!keep_constant_thr){
+                current_throttle = Map.throttle(busData[ELRS_THROTLE]);
+            }
 
-            _packet.throttle = Map.throttle(busData[ELRS_THROTLE]);
+            _packet.throttle = current_throttle*Map.throttle_ratio(busData[ELRS_RATIO_THROTTLE]);
             _packet.brake = 0.0; // TODO: (Moises) Implement brake
```

Comment:
Add the brake implementation back.

Reply:
Not possible, how?

Reply:
Ok understood. I thought you had one from before on the RC controller.